src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/HMACSHA512.cs

```diff
@@ -40,11 +40,9 @@ public bool ProduceLegacyHmacValues
             }
             set
             {
-                // We don't have a managed implementation of HMAC
-                // so we can't support this
                 if (value)
                 {
-                    throw new PlatformNotSupportedException();
+                    throw new PlatformNotSupportedException(); // We don't have a managed implementation of HMAC so we can't support this 
```

Comment:
The property (ProduceLegacyHmacValues) means "produce the wrong answer that HMACSHA512 gave on .NET 2.0". The native implementations never had that bug, so they don't have that quirk.